.circleci/config.yml

```diff
@@ -209,6 +217,10 @@ commands:
         description: "The gold-standard to use for generating the answers"
         type: string
         default: "gold-standard-nccv1"
+      cache-tag:
```

Comment:
We need to add a small unique id to the cache name because once a cache is uploaded it can't be overwritten. We do this in case we make changes to the gold standard answers without bumping the standard number. For example, this may happen in the PR that bumps the gold standard if there is a little juggling before the PR is accepted.